libraries/postgres/src/Relations.js

```diff
@@ -0,0 +1,53 @@
+const {isPlainObject} = require('lodash');
+const assert = require('assert').strict;
+
+class Relations {
```

Comment:
What are relations? A comment here perhaps would help.

Reply:
Will do -- I'm expecting that this will be useful for a `views.yml` too, someday, and "relation" is just a general term for anything with rows and columns, such as tables or views.  I'll add a comment.